packages/beacon-node/test/unit/network/reqresp/utils.ts

```diff
@@ -37,6 +37,22 @@ export function generateEmptySignedBlocks(n = 3): phase0.SignedBeaconBlock[] {
   return Array.from({length: n}).map(() => generateEmptySignedBlock());
 }
 
+export function generateDefaultLightClientBootstrap(): altair.LightClientBootstrap {
+  return ssz.altair.LightClientBootstrap.defaultValue();
+}
+
+export function generateDefaultLightClientFinalityUpdate(): altair.LightClientFinalityUpdate {
+  return ssz.altair.LightClientFinalityUpdate.defaultValue();
+}
+
+export function generateDefaultLightClientOptimisticUpdate(): altair.LightClientOptimisticUpdate {
+  return ssz.altair.LightClientOptimisticUpdate.defaultValue();
+}
+
+export function generateDefaultLightClientUpdate(): altair.LightClientUpdate[] {
```

Comment:
Are these helpers really necessary? Why not just write `[ssz.altair.LightClientUpdate.defaultValue()]` in the consumer code?